src/Console/PoToPhp.php

```diff
@@ -103,14 +103,14 @@ protected function doExecute()
 
 			if (substr($l, 0, 15) == '"Plural-Forms: ') {
 				$match = [];
-				preg_match("|nplurals=([0-9]*); *plural=(.*)[;\\\\]|", $l, $match);
-				$cond = str_replace('n', '$n', $match[2]);
+				preg_match("|nplurals=([0-9]*); *plural=(.*?)[;\\\\]|", $l, $match);
+				$return = $this->convertCPluralConditionToPhpReturnStatement($match[2]);
```

Comment:
Does this win the price of the longest function name you ever created? :grimacing:

Reply:
Possibly, although it's an open contest.